src/components/ParticipantList/ParticipantList.js

```diff
@@ -37,7 +37,8 @@ class ParticipantList extends Component {
   };
 
   handleInput = object => {
-    this.props.collectData(object);
+    this.props.saveParticipant(object);
+    console.log(this.props.participants);
```

Comment:
`console.log` can be used for development in personal branch, but should not be merged to master